Makefile

```diff
@@ -5,6 +5,14 @@ GOOS=linux
 CORE_IMAGES=./cmd/bash ./cmd/controller ./cmd/entrypoint ./cmd/gsutil ./cmd/kubeconfigwriter ./cmd/nop ./cmd/webhook ./cmd/imagedigestexporter
 CORE_IMAGES_WITH_GIT=./cmd/creds-init ./cmd/git-init
 
+##
+# You need to provide a RELEASE_VERSION when using targets like `push-image`, you can do it directly
+# on the command like this: `make push-image RELEASE_VERSION=0.4.0`
+RELEASE_VERSION=
+REGISTRY_CI_URL=registry.svc.ci.openshift.org/openshift/tektoncd-v$(RELEASE_VERSION):tektoncd-pipeline
+REGISTRY_RELEASE_URL=quay.io/openshift-pipeline/tektoncd-pipeline
+BINARY_WITHOUT_IMAGE=gsutil # TODO: why those are not in CI ?
```

Comment:
This won't be needed anymore with #40

Reply:
But is the pushing image from private ci to public quay needed in the future?